app/models/grda_warehouse/tasks/push_clients_to_cas.rb

```diff
@@ -67,6 +67,10 @@ def sync!
                 :source_eccovia_case_managers,
               ]
             end
+            safe_project_names = {}
+            GrdaWarehouse::Hud::Project.all.each do |project|
```

Comment:
Small memory tweak:
```suggestion
            GrdaWarehouse::Hud::Project.find_each do |project|
```

Reply:
I will have this included in the upcoming commit